bot/controllers/welcome.js

```diff
@@ -0,0 +1,33 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+// const util = require('apex-util');
```

Comment:
What is the purpose of commenting out the debugger?

Reply:
The purpose was because I did not have a need for it anymore. I should have deleted the line instead of commenting it out though